imager_images/models.py

```diff
@@ -43,7 +43,7 @@ class Photo(models.Model):
                                  choices=[('private', 'private'),
                                           ('shared', 'shared'),
                                           ('public', 'public')])
-    likes_cheese = models.BooleanField('Likes Cheese!', default=False)
+    is_public = models.BooleanField('Is_public?', default=False)
 
     def __str__(self):
         '''this is a  doc string'''
```

Comment:
Worth noting that this `__str__` method is supposed to return a `self.image_file` but that field doesn't actually exist amongst the ones you've declared above.